# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/rpm/link-lua.patch
# Copyright (C) 2021 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- rpm-4.17.0/rpmio/Makefile.am.vanilla	2021-10-07 16:42:13.133000000 +0200
+++ rpm-4.17.0/rpmio/Makefile.am	2021-10-07 16:42:23.977000000 +0200
@@ -33,7 +33,7 @@
 endif
 endif
 
-librpmio_la_LDFLAGS = -version-info $(rpm_version_info)
+librpmio_la_LDFLAGS = -llua -version-info $(rpm_version_info)
 librpmio_la_LIBADD = \
 	../misc/libmisc.la \
 	@WITH_OPENSSL_LIB@ \
